doc/whats-new.rst

```diff
@@ -109,6 +111,9 @@ Breaking changes
   ``ds.coarsen(...).mean(keep_attrs=False)`` instead of ``ds.coarsen(..., keep_attrs=False).mean()``.
   Further, coarsen now keeps attributes per default (:pull:`5227`).
   By `Mathias Hauser <https://github.com/mathause>`_.
+- switch the default of the :py:func`merge` ``combine_attrs`` parameter to ``"override"``
+  (:pull:`4902`)
+  By `Justus Magin <https://github.com/keewis>`_.
```

Comment:
this didn't go through a deprecation cycle, but I'm not sure we could have done much to fix this, especially since we're planning to switch the default for `keep_attrs` / `combine_attrs` to `drop_conflicts`.